utils/vlog/tests/muxes/golden.pb_type.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version='1.0' encoding='utf-8'?>
+<pb_type xmlns:xi="http://www.w3.org/2001/XInclude" name="use_mux" num_pb="1">
+  <input name="a" num_pins="1"/>
+  <input name="b" num_pins="1"/>
+  <input name="c" num_pins="1"/>
+  <output name="o1" num_pins="1"/>
+  <output name="o2" num_pins="1"/>
+  <interconnect>
+    <mux name="mux1">
```

Comment:
The xmlsort.xsl script transforms the output into;
```xml
    <mux name="mux1" input="use_mux.a use_mux.b " output="use_mux.o1 ">
      <metadata>
        <meta key="fasm_mux">
            use_mux.a : I0
            use_mux.b : I1
        </meta>
        <meta key="type">bel</meta>
        <meta key="subtype">routing</meta>
      </metadata>
    </mux>
```

Reply:
Can we get a diff test on that behavior please?

Reply:
This was extracted into https://github.com/SymbiFlow/symbiflow-arch-defs/pull/728